src/View/Home/event.html.twig

```diff
@@ -0,0 +1,211 @@
+{% extends 'layout.html.twig' %}
+{% block title %}Les Temps des Loisirs | Évènements
+{% endblock %}
+{% block stylesheet %}<link rel="stylesheet" href="/assets/css/event.css">
+	<link rel="stylesheet" href="/assets/css/scroll.css">
+	<link rel="stylesheet" href="/assets/css/admin.css">
+{% endblock %}
+
+{% block content %}
+	<main class="container-fluid p-0 eventMain">
+
+
+		<h1 class="text-uppercase text-center mb-5 mt-5">Tous Nos évènements</h1>
+		<div class="bg-eventCards">
+			<section class="container d-flex justify-content-between flex-wrap">
+
+				<div class="flip-card">
+
+					<div class="flip-card-inner">
+						<div class="flip-card-front bg-front-1">
+							<h3>Bingo</h3>
+
+						</div>
+						<div class="flip-card-back">
+							<h3>Bingo</h3>
+							<p>On fait de bingo reguliérment entre les adhérents et ses familles. Ce jeu, permettant de gagner des prix est très populaire entre notres adhérents.
+							</p>
+						</div>
+					</div>
+				</div>
+
+				<div class="flip-card">
+					<div class="flip-card-inner">
+						<div class="flip-card-front bg-front-2">
+							<h3>Voyages</h3>
+						</div>
+						<div class="flip-card-back">
+							<h3>Voyages</h3>
+							<p>Le temps des loisirs a pour but de regrouper les passionnés de voyage afin de favoriser, entre les adhérents, les échanges de vécu et d'informations pratiques sur leurs voyages, de mettre en commun ces éléments, et de faciliter ainsi la préparation d'un prochain voyage.
+
+							</p>
+						</div>
+					</div>
+				</div>
+
+				<div class="flip-card">
+					<div class="flip-card-inner">
+						<div class="flip-card-front bg-front-3">
+							<h3>Sorties</h3>
+						</div>
+						<div class="flip-card-back">
+							<h3>Sorties</h3>
+							<p>Nous favorisons des départs réguliers parmi les membres et les sorties sont ouvertes à tous les adhérents, cependant certaines d'entre elles sont plus particulièrement destinées aux débutants.
+							</p>
+						</div>
+					</div>
+				</div>
+
+			</section>
+		</div>
+
+		<section class="container text-uppercase text-center mb-5 mt-5">
+			<h2 class="m-5">Calandrier</h2>
+
+			<div
+				class="currentDateTitle">
+
+				{#To be changed dynamically #}
+				<h4
+					class="m-4 p-2">
+					
+					{% if 'now'|date("m") == "01"%}
```

Comment:
le code est ok mais n'est pas DRY.
Essaye plutot 
- de créer un tableau avec tes 12 noms de mois (tu peux le faire direct coté Twig avec {% set %}
- tu boucles sur le tableau en ajoutant ta condition
- pas besoin d'attribut sytle, utilise une classe

Reply:
Done! Tks for the tip!